Add sports complex based queries and response

rakki-18 opened the pull request:
Added examples based on queries related to going to sports complex.
Added a utter_go_to_sports_complex response and a sports_complex intent.

rakki-18 committed:
Add sports complex queries and response

Added queries based on sports complex and added a go_to_sports_complex response.

rakki-18 committed:
Fixed typo in README

Fixed a small typo in README :)

rakki-18 committed:
Change Academic_building to sports_complex

Fixed a typo.

adityasohoni commented:
Seems good. You can add more examples to the data. Also since the user may not know where the sports complex is, add a more appropriate message.

rakki-18 committed:
Add more examples and a detailed response

More examples were added for sports_complex and a detailed response to go to sports_complex was added.

rakki-18 commented:
@adityasohoni I have added more examples and a detailed response for sports complex.

adityasohoni commented:
Nice, I will be merging this.
You can now take up one of the other 'good first issues' now or add another functionality.
If you have any other suggestions you can discuss them too.
